pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/V1Constants.java

```diff
@@ -44,14 +44,17 @@ public static class Indexes {
     public static final String UNSORTED_MV_FORWARD_INDEX_FILE_EXTENSION = ".mv.fwd";
     public static final String BITMAP_INVERTED_INDEX_FILE_EXTENSION = ".bitmap.inv";
     public static final String BITMAP_RANGE_INDEX_FILE_EXTENSION = ".bitmap.range";
-    public static final String FST_INDEX_FILE_EXTENSION = ".lucene.fst";
     public static final String JSON_INDEX_FILE_EXTENSION = ".json.idx";
     public static final String NATIVE_TEXT_INDEX_FILE_EXTENSION = ".nativetext.idx";
     public static final String H3_INDEX_FILE_EXTENSION = ".h3.idx";
     public static final String BLOOM_FILTER_FILE_EXTENSION = ".bloom";
     public static final String NULLVALUE_VECTOR_FILE_EXTENSION = ".bitmap.nullvalue";
+    public static final String FST_INDEX_FILE_EXTENSION = ".lucene.fst";
```

Comment:
Rename it to `LUCENE_FST_INDEX_FILE_EXTENSION`. Also seems it is mistakenly used as native FST extension as well, so you might need to take a look if that needs to be fixed

Reply:
right, I think it's better to have a separated pr for this issue.